spark-plugins/docs/DecisionTreeTrainer-sparksink.md

```diff
@@ -24,7 +24,10 @@ fields cannot be specified.
 featureFieldsToInclude and featureFieldsToExclude are empty, all fields except the label will be used for training. Both
  *featureFieldsToInclude* and *featureFieldsToExclude* fields cannot be specified.
 
-**cardinalityMapping:** Mapping of the feature to the cardinality of that feature; required for categorical features.
+**arityMapping:** List of the categorical features along with its arity value, where arity indicates the number
```

Comment:
Changed the ``cardinalityMapping`` to ``arityMapping``, to match the description.
Since the value entered(arity) for this configuration indicates the number of arguments a particular categorical feature can take. Its not exactly related to cardinality.

Should we change the name back to 'cardinalityMapping' for this parameter?